regolith/validators.py

```diff
@@ -1,10 +1,15 @@
-"""Validators and convertors for regolith input."""
+"""Validators and converters for regolith input."""
+import collections
 import os
 import re
+
+from cerberus import Validator
 from getpass import getpass
```

Comment:
PEP8 cerberus is a 3rd party library and so should go below getpass, which is from the standard library and should be grouped with those above.